packages/aft/lib/src/commands/bootstrap_command.dart

```diff
@@ -70,6 +70,8 @@ const amplifyconfig = \'\'\'{
   "UserAgent": "aws-amplify-cli/2.0",
   "Version": "1.0"
 }\'\'\';
+
+const amplifyEnvironments = <String, String>{};
```

Comment:
Another option would be to put each config it a unique file, although I am not sure either way is clearly better than the other. 

The `bootstrap` command would have to know what environments exist so that it could create the stubs. That could be done via an aft config for each project (aft_config.yaml or something).

I think this is fine though.

Reply:
Yeah, it's tricky. My thinking with this approach is that it could tie nicely into the Amplify CLI's environment flow. The `amplifyEnvironments` variable would always have all the environments while the `amplifyconfig` variable would point to the currently checked-out one. So, ideally, your code would never break and you could always use either the current environment (`amplifyconfig`) or a specific one (`amplifyEnvironments`).

Not sure if it's the best yet, but I think it could work.